### PR TITLE
PR: Enable more qt6 tests

### PR DESCRIPTION
**Pull request by jschueller**
the disabled classes QCameraViewfinder, QSGAbstractRenderer& QSGEngine are qt5-only

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -79,7 +79,7 @@ jobs:
         shell: bash
         run: |
           sudo apt update
-          sudo apt install libpulse-dev libegl1-mesa libopengl0
+          sudo apt install libpulse-dev libegl1-mesa libopengl0 gstreamer1.0-plugins-base gstreamer1.0-gl
```
Comment (dalthviz) [inline .github/workflows/ci.yml:82]: I think this is needed for `test_multimedia.py` but I think no changes have been done to that test file. Maybe this is needed for some other test? If not then I think we should revert this change

File: qtpy/tests/test_qtmultimediawidgets.py [hunk 2]
```diff
@@ -6,15 +6,11 @@
...
-@pytest.mark.skipif(not (PYQT5 or PYSIDE2), reason="Only available in Qt5 bindings")
-@pytest.mark.skipif(
-    using_conda(), reason="Conda packages don't seem to include QtMultimedia"
-)
 def test_qtmultimediawidgets():
     """Test the qtpy.QtMultimediaWidgets namespace"""
     from qtpy import QtMultimediaWidgets
 
-    assert QtMultimediaWidgets.QCameraViewfinder is not None
+    # assert QtMultimediaWidgets.QCameraViewfinder is not None
```
Comment (dalthviz) [inline qtpy/tests/test_qtmultimediawidgets.py:13]: I think it would be better to add a validation to test this instead of just commenting the assert, something like:

```python
if QT5:    # Only available for Qt5 bindings
    assert QtMultimediaWidgets.QCameraViewfinder is not None
```
Probably the `QtMultimediaWidgets.QVideoWidgetControl` was commented for that reason and we could move it to the Qt5 validation?

File: qtpy/tests/test_qtquick.py [hunk 3]
```diff
@@ -32,11 +31,11 @@ def test_qtquick():
     assert QtQuick.QQuickTextureFactory is not None
     assert QtQuick.QQuickView is not None
     assert QtQuick.QQuickWindow is not None
-    assert QtQuick.QSGAbstractRenderer is not None
+    # assert QtQuick.QSGAbstractRenderer is not None
     assert QtQuick.QSGBasicGeometryNode is not None
     assert QtQuick.QSGClipNode is not None
     assert QtQuick.QSGDynamicTexture is not None
-    assert QtQuick.QSGEngine is not None
+    # assert QtQuick.QSGEngine is not None
```
Comment (dalthviz) [inline qtpy/tests/test_qtquick.py:38]: I think it would be better to add here too a validation to test these modules instead of just commenting the asserts, something like:

```python
if QT5:    # Only available for Qt5 bindings
    assert QtQuick.QSGAbstractRenderer is not None
    assert QtQuick.QSGEngine is not None
```

**Review by dalthviz — COMMENTED** [review]
Thanks @jschueller for working on this! I left some comments/suggestions regarding the commented asserts for modules only available on Qt5 bindings. Also, left a question regarding the installation of GStreamer for Linux.

Other than that this LGTM :+1:

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jschueller) [reply .github/workflows/ci.yml:82]: this is needed for the pyqt6 wheels

File: qtpy/tests/test_qtquick.py (hunk 3 above)
Comment (jschueller) [reply qtpy/tests/test_qtquick.py]: ok

File: qtpy/tests/test_qtmultimediawidgets.py (hunk 2 above)
Comment (jschueller) [reply qtpy/tests/test_qtmultimediawidgets.py]: actually QVideoWidgetControl seems not available with pyqt5

**Review by dalthviz — APPROVED** [review]
Thanks @jschueller !
